src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1648,8 +1635,8 @@ private function registerMessengerConfiguration(array $config, ContainerBuilder
 
         $senderAliases = [];
         foreach ($config['transports'] as $name => $transport) {
-            if (0 === strpos($transport['dsn'], 'amqp://') && !$container->hasDefinition('messenger.transport.amqp.factory')) {
-                throw new LogicException('The default AMQP transport is not available. Make sure you have installed and enabled the Serializer component. Try enabling it or running "composer require symfony/serializer-pack".');
```

Comment:
This also doesn't seem to have made any sense - the amqp factory is always there (see comment above) and it makes no sense to throw an exception about installing the serializer if it was not. I think both of these things are leftover from some earlier version.

Reply:
TBH, these lines have always been a pain so if we can remove them, that's great. Because the default is now using `serialize`, I think it indeed makes sense 👍